server/controllers/users-controller.js

```diff
@@ -0,0 +1,40 @@
+import { successResponse, generateToken, errorResponse } from '../utils/helpers';
+import { User } from '../models';
+
+/**
+ * The controllers for users route
+ *
+ * @class UsersController
+ */
+class UsersController {
```

Comment:
`UsersController` should not be pluralized.
The handler methods contained within should be linked to all user related routes

Reply:
Pluralization (or lack of) is pretty much a matter of preference here.